### DISCUSSION TITLE
[Feature] Option to change `<script lang="ts" setup>` to `<script setup lang="ts">`

### DISCUSSION THREAD

**Discussion by larrasu** [nuxtrdev/nuxtr-vscode#137] [Ideas]
I know this is for purely code aesthetics only and doesn't affect pragmatically. But is this possible? I always edit the tag after creating a new file.

**Comment by adhamfarrag** [comment]
Hi @larrasu, 

[File Templates](https://github.com/nuxtrdev/nuxtr-vscode?tab=readme-ov-file#personalized-vue-file-templates) feature offers a way to have your own templates, building default templates will automatically scaffold new files using your preferred changes. You can create one either from the sidebar or just create a page/layout and right click to `Create Vue Page/Layout template`. 

I'm working on a major release and it should include the same thing for components and pretty much all Nitro files too.
